specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/gallery.json

```diff
@@ -2621,6 +2762,57 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "SharingUpdate": {
+      "description": "Specifies information about the gallery sharing profile update.",
+      "properties": {
+        "operationType": {
+          "type": "string",
+          "description": "This property allows you to specify the operation type of gallery sharing update. <br><br> Possible values are: <br><br> **Add** <br><br> **Remove** <br><br> **Reset**",
+          "enum": [
+            "Add",
+            "Remove",
+            "Reset"
+          ],
+          "x-ms-enum": {
+            "name": "SharingUpdateOperationTypes",
+            "modelAsString": false
+          }
```

Comment:
should operation type be required?

Reply:
groups as well?

Reply:
groups do not need to be required considering the 'reset' case.